### PR TITLE
Fixes food duplication bug, adds making batches of candies and cooked goods.

### PR DESCRIPTION
**Pull request by Iamgoofball**

### CODE REVIEW

**Comment by Atskadan** [comment]
:-1: no bakers dozen

**Comment by RemieRichards** [comment]
Something tells me this shouldn't have been merged.

File: code/game/machinery/kitchen/oven.dm [hunk 1]
```diff
@@ -23,26 +23,34 @@
...
-			sleep(100)
+			sleep(50*batchsize)
 			on = FALSE
 			if(!candy)
 				icon_state = "oven_off"
 			else
 				icon_state = "mixer_off"
+			for(var/i = 0, i < batchsize, i++)
+				var/obj/item/weapon/reagent_containers/food/snacks/customizable/B = C
+				B.loc
```
Comment (RemieRichards) [inline code/game/machinery/kitchen/oven.dm:53]: This. what the fuck is this supposed to do.
B.loc. you're stating B.loc and then doing nothing with it.

**Comment by Iamgoofball** [comment]
SHIT WAIT I FORGOT TO PUT A WIP ON THIS FUCK FUCK FUCK

worst coder 2557

**Comment by RemieRichards** [comment]
@dannno don't worry I already emergency fixed goof's slipup

**Pull request merged**